setup.py

```diff
@@ -28,123 +30,54 @@
 #
 # =================================================================
 
+import io
 import os
-from distutils.core import setup
-import pycsw
 
+from setuptools import find_packages, setup
 
-def is_package(path):
-    """Decipher whether a filepath is a Python package"""
-    return (
-        os.path.isdir(path) and
-        os.path.isfile(os.path.join(path, '__init__.py'))
-    )
 
+def read(*names, **kwargs):
+    return io.open(
```

Comment:
OK I'll use `with` then.It seemed pretty innocuous not to use it here (sincewe're reading our own files) but I guess you're right ;)